amp-conformance/run_tests.pl.in

```diff
@@ -30,12 +32,13 @@ use File::Basename;
 use File::Find;
 use Getopt::Long;
 use Safe;
+use strict;
 use threads;
 use threads::shared;
-use strict;
+use warnings;
 
 my $max_num_threads = @AMP_CONFORMANCE_NUM_THREADS@;
-my $use_threads = $Config{useithreads} and $max_num_threads > 1;
+my $use_threads = $Config{useithreads};# && $max_num_threads > 1; - temporarily disabled.
```

Comment:
@AlexVlx could you explain why this should be disabled? it would greatly enlarge time required for completing C++AMP conformance test.